lib/config.js

```diff
@@ -1,44 +1,51 @@
-var defaults = require('lodash.defaults');
-var compact = require('lodash.compact');
-var map = require('lodash.map');
-var fs        = require('fs');
-var path      = require('path');
-var osHomedir = require('os-homedir');
+'use strict';
 
-var config = null;
+const defaults = require('lodash.defaults');
```

Comment:
Why would I be opposed ? :laughing: Unless something automatically does this, I would actually unalign them since that's the natural way of writing. Will take care of it in a later PR.